cli/packages/cmd/login.go

```diff
@@ -252,6 +274,51 @@ func init() {
 	rootCmd.AddCommand(loginCmd)
 }
 
+func askForDomain() error {
+	//query user to choose between Infisical cloud or self hosting
+	options := []string{"Infisical Cloud", "Self Hosting"}
+	optionsPrompt := promptui.Select{
+		Label: "Select your hosting option",
+		Items: options,
+		Size:  2,
+	}
+
+	idx, _, err := optionsPrompt.Run()
+	if err != nil {
+		return err
+	}
+
+	if idx == 0 {
+		//cloud option
+		config.INFISICAL_URL = util.INFISICAL_DEFAULT_API_URL
+		return nil
+	}
+
+	urlValidation := func(input string) error {
+		_, err := url.ParseRequestURI(input)
+		if err != nil {
+			return errors.New("this is an invalid url")
+		}
+		return nil
+	}
+
+	//else run prompt to enter domain
+	domainPrompt := promptui.Prompt{
+		Label:    "Domain",
+		Validate: urlValidation,
+	}
+
+	domain, err := domainPrompt.Run()
+	if err != nil {
+		return err
+	}
+
+	//set api url
+	config.INFISICAL_URL = domain
```

Comment:
we should add a `/api` at the end because all of our default nginx configs route traffic from `/api` to the backend services

Reply:
Yeah, my initial thought was that will be added directly by the users. However adding a placeholder text example you suggested, as to the format they're to input would resolve the ambiguity.

Reply:
yes, good thought. Let's add the /api in the place holder text so they know